src/react/account/AccountUserList.tsx

```diff
@@ -150,7 +151,7 @@ const AccountUserList = ({ accountName }: { accountName?: string }) => {
 
   const listUsersQuery = useAwsPaginatedEntities(
     {
-      queryKey: ['listIAMUsers', accountName],
+      queryKey: getUserListUsersQuery(accountName, notFalsyTypeGuard(IAMClient)).queryKey,
       queryFn: (_ctx, marker) => {
```

Comment:
It should be directly `const listUsersQuery = useAwsPaginatedEntities(getUserListUsersQuery(accountName, notFalsyTypeGuard(IAMClient)));`